src/cards/arrow.ts

```diff
@@ -48,19 +48,14 @@ const spell: Spell = {
     }
   }
 };
-export function arrowEffect(multiShotCount: number, collideFnKey: string, doesPierce: boolean = false) {
+export function arrowEffect(multiShotCount: number, collideFnKey: string, piercesRemaining: number = 0, bouncesRemaining: number = 0) {
   return async (state: EffectState, card: ICard, quantity: number, underworld: Underworld, prediction: boolean, outOfRange?: boolean) => {
     let targets: Vec2[] = state.targetedUnits;
-    const path = findArrowPath(state.casterPositionAtTimeOfCast, state.castLocation, underworld)
-    targets = targets.length ? targets.map(t => {
-      const path = findArrowPath(state.casterPositionAtTimeOfCast, t, underworld);
-      return path ? path.p2 : state.castLocation;
-    }) : [path ? path.p2 : state.castLocation];
```

Comment:
Was this already unused? I don't see any reference to it

Reply:
> @jdoleary
> Was this already unused?

In the main `arrowEffect` function, `findArrowPath()` was used to pre-determine an "endpoint" for the arrow, so that forcemoves could remove it when it hit a wall, but now that's done via physics collision. `findArrowPath()` was also responsible for setting Target Arrow's cast location and preventing weird interactions like placing a Target Circle out of range/on the other side of a wall, but this is also handled through collision now